templates/repo/settings/options.tmpl

```diff
@@ -976,20 +976,23 @@
 				{{end}}
 			</div>
 			<div class="content">
-				<p>
+				<div class="ui warning message">
```

Comment:
It makes the UI looks unnecessarily complex by using such `warning message`.

I never see a dialog with inner warning color/border.